gremlin-go/driver/performance-tests/java-test/src/main/java/PerformanceTest.java

```diff
@@ -0,0 +1,165 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+// CAUTION!!!
+// This file is intended for generating Performance test results only.
+// It is not to be used in a production setting.
+// It is being made available as a reference to the test results for
+// anyone that wants see the code used to obtain the performance metrics.
+
+import org.apache.tinkerpop.gremlin.driver.Cluster;
+import org.apache.tinkerpop.gremlin.driver.remote.DriverRemoteConnection;
+import org.apache.tinkerpop.gremlin.process.traversal.dsl.graph.GraphTraversal;
+import org.apache.tinkerpop.gremlin.process.traversal.dsl.graph.GraphTraversalSource;
+import org.apache.tinkerpop.gremlin.process.traversal.dsl.graph.__;
+import java.time.Duration;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import static org.apache.tinkerpop.gremlin.process.traversal.AnonymousTraversalSource.traversal;
+
+public class PerformanceTest {
+    public static void main(String[] args) {
+        executePerformanceTests();
+        System.exit(0);
+    }
+    static final int SAMPLE_SIZE = 11;
+    static final int VALUE_MAP_REPEATS = 500;
+    static final Cluster cluster = Cluster.build("172.31.24.17").port(45940).maxContentLength(Integer.MAX_VALUE).create();
```

Comment:
```suggestion
    static final Cluster cluster = Cluster.build("172.31.24.17").port(45940).maxContentLength(314572800).create();
```
300MB is a good working number (locally for sure), it is what I'm using for buffer sizes in go.

Reply:
To add to Yang's suggestion, I suggest using `300 * 1024 * 1024` for 300 MB, that was it is clear when you look at it what it is. 314572800 is a little confusing when you look at it.